fieldservice_maintenance/models/fsm_order.py

```diff
@@ -10,31 +10,3 @@ class FSMOrder(models.Model):
 
     request_id = fields.Many2one('maintenance.request',
                                  string='Maintenance Request')
-
-    # @api.model
-    # def create(self, vals):
-    #     # if FSM order with type maintenance is create then
-    #     # create maintenance request
-    #     order = super(FSMOrder, self).create(vals)
-    #     if order.type == 'maintenance':
-    #         employee_rec = self.env['hr.employee'].search(
-    #             [('user_id', '=', self.env.uid)], limit=1)
-    #         if order.equipment_id and not order.request_id:
-    #             equipment = order.equipment_id
-    #             team_id = equipment.maintenance_equipment_id and\
-    #                 equipment.maintenance_equipment_id.maintenance_team_id.id
-    #             request_id = self.env['maintenance.request'].with_context(
-    #                 fsm_order=True).create({
-    #                     'name': order.name or '',
-    #                     'employee_id': employee_rec.id,
-    #                     'equipment_id':
-    #                         equipment.maintenance_equipment_id.id,
-    #                     'category_id': equipment.category_id.id,
-    #                     'request_date': fields.Date.context_today(self),
-    #                     'maintenance_type': 'corrective',
-    #                     'maintenance_team_id': team_id,
-    #                     'schedule_date': order.request_early,
-    #                     'description': order.description
-    #                 })
-    #             order.request_id = request_id
-    #     return order
```

Comment:
@bodedra We need to reactivate this method to create the maintenance request.

Reply:
@max3903 This method is already in comment. I just remove commented code.

If our goal to uncomment this method, then `hr` module is a valid as a dependency.

Reply:
We can remove the employee_id field and the dependency on hr.

Reply:
Sure. Will do it.